cft/deploy.yml

```diff
@@ -0,0 +1,38 @@
+Resources:
+  ElasticsearchDomain:
+    Type: AWS::Elasticsearch::Domain
+    Properties:
```

Comment:
Coverity found issue: Insecure TLS version - CWE-327, Low Severity

****

**How to fix:** Configure the ElasticSearch service domain to require TLS 1.2 or newer by explicitly setting the `Properties.DomainEndpointOptions.TLSSecurityPolicy` value to `Policy-Min-TLS-1-2-2019-07`.

<details>
<summary>Click to expand data flow...</summary>

**From cft/deploy.yml:**
```
    1 Resources:
    2   ElasticsearchDomain:
    3     Type: AWS::Elasticsearch::Domain
1. Sigma main event: The Amazon Elasticsearch Service (Amazon ES) domain supports TLS 1.0.

2. remediation: Configure the ElasticSearch service domain to require TLS 1.2 or newer by explicitly setting the `Properties.DomainEndpointOptions.TLSSecurityPolicy` value to `Policy-Min-TLS-1-2-2019-07`.

    4     Properties:
    5       DomainName: "test"
    6       ElasticsearchClusterConfig:
```
</details><!-- Coverity 26d2be62f414fa8f364371120dfb214c -->